tests/testthat/test-moduleCohortRelationship.R

```diff
@@ -0,0 +1,52 @@
+test_that("Testing cohort relationship logic", {
+  # manually create cohort table and load to table
+  
+  # connection <-
```

Comment:
I think you want to create a connection with the `connectionDetails` object created in `setup.R`.

I've also been using `with_dbc_connection` in the helpers.R utilities to ensure that the connection closes.

For example

```{r}
connection <- DatabaseConnector::connect(connectionDetails)
with_dbc_connection(connection, {
   ...

  runCohortRelationshipDiagnostics(
      connection = connection,
      cohortDatabaseSchema = NULL,
      cdmDatabaseSchema = cdmDatabaseSchema,
      cohortTable = "#cohort_relationship",
      targetCohortIds = c(1, 2, 3),
      comparatorCohortIds = c(10, 11, 12),
      relationshipDays = dplyr::tibble(startDay = temporalStartDays,
                                       endDay = temporalEndDays)
  )

  ...
}
```